packages/ui/react-ui-stack/src/components/props.ts

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import type { FC, PropsWithChildren } from 'react';
+
+import type { MosaicActiveType, MosaicContainerProps, MosaicDataItem, MosaicTileProps } from '@dxos/react-ui-mosaic';
+
+export type StackSectionContent = MosaicDataItem & { title?: string };
```

Comment:
skeptical about factoring out all props to a single file away from actual source/usage, maybe an antipattern? what's the motivation here?

Reply:
This was to prevent circular dependencies between Stack and its Sections, since they’re in different files. I only move types to a common file when genuinely necessary, will reassess here since things have changed since your comment.

Reply:
Was able to revert 👍